### PR TITLE
ci: notify tools repo when there is a new release

### PR DESCRIPTION
**Pull request by craicoverflow**
Sends a workflow dispatch event to https://github.com/redhat-developer/app-services-tools to initiate a new image release to Quay.

There is no way to verify this until our next release.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -72,4 +72,21 @@ jobs:
...
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Notify tools repo
+        uses: actions/github-script@v6
+        with: 
+          script: |
+            github.request('POST /repos/redhat-developer/app-services-tools/actions/workflows/20880234/dispatches', {
+              workflow_id: 20880234,
```
Comment (craicoverflow) [inline .github/workflows/release.yml:87]: Workflow ID from here: https://api.github.com/repos/redhat-developer/app-services-tools/actions/workflows/20880234

**Review by wtrocki — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (wtrocki) [reply .github/workflows/release.yml:87]: Better way of triggering jobs?

Comment (craicoverflow) [reply .github/workflows/release.yml:87]: it is different to repository dispatch, not sure if it is objectively better or not. It is the reason we can trigger manually with a form and also can be triggered on non-main branches so it is a bit more flexible.

Comment (wtrocki) [reply .github/workflows/release.yml:87]: Reason I'm asking is that I would need to setup another repo for that today and was wondering which job to use. 
Will try this now. Thank you

**Pull request merged**